src/commands/createFunctionApp/FunctionAppRuntimeStep.ts

```diff
@@ -50,7 +52,7 @@ export class FunctionAppRuntimeStep extends AzureWizardPromptStep<IFunctionAppWi
                 picks.unshift({ label: 'Node.js 8.x', data: 'node|8' });
             }
             picks.unshift({ label: 'Node.js 10.x', data: 'node|10' });
-            if (context.version === FuncVersion.v3) {
+            if (context.version !== FuncVersion.v2) {
                 picks.unshift({ label: 'Node.js 12.x', data: 'node|12' });
```

Comment:
Node.js 12 is available on v1?

Reply:
Nope. We're in an `else` case of `if (context.version === FuncVersion.v1) {` from above. I did this because if/when there's a v4, it's better to assume it will support Node 12